Author: Steve Beattie <steve.beattie@canonical.com>
Description: enable bind now by default when linking with pie by default

---
 gcc/gcc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/gcc/gcc.c
===================================================================
--- a/gcc/gcc.c
+++ b/gcc/gcc.c
@@ -930,7 +930,7 @@ proper position among the other output f
 #ifndef LINK_PIE_SPEC
 #ifdef HAVE_LD_PIE
 #ifndef LD_PIE_SPEC
-#define LD_PIE_SPEC "-pie"
+#define LD_PIE_SPEC "-pie -z now"
 #endif
 #else
 #define LD_PIE_SPEC ""
